src/styles.css

```
@@ -694,6 +694,11 @@ body {
 
 :root {
   --datacite-color-primary: #243b54;
+  --datacite-color-secondary: #1abc9c;
+  --datacite-color-secondary: red;
```

Stylelint (17.9.0)

[error] Duplicate custom property "--datacite-color-secondary" (declaration-block-no-duplicate-custom-properties)

(declaration-block-no-duplicate-custom-properties)